odoo/addons/base/tests/test_ir_http.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import re
+import time
+
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('-at_install', 'post_install')
+class TestIrHttpPerformances(TransactionCase):
+
+    def test_routing_map_performance(self):
+        self.env['ir.http']._clear_routing_map()
+        # if the routing map was already generated it is possible that some compiled regex are in cache.
+        # we want to mesure the cold state, when the worker just spawned, we need to empty the re cache
+        re._cache.clear()
+
+        self.env['ir.http']._clear_routing_map()
+        start = time.time()
```

Comment:
should had been [time.perf_counter](https://docs.python.org/3/library/time.html#time.perf_counter), time.time is subject to system clock changes, perf_counter isn't, this is minor and it's too late, meh :shrug:

Reply:
I don't think the system clock will change during this test  :D

In any case there are multiple places where we are using that i copied this from another test, we can adapt all of them but I would like to have this merged asap and reach master to rebase my other branch, we can change that later :)